Arduino.mk

```diff
@@ -1039,9 +1045,22 @@ ifndef AR_NAME
     endif
 endif
 
+ifeq ($(shell expr $(CC_VERNUM) '>' 490), 1)
```

Comment:
seems like the fedora bug should just be documented and users told to set USE_LTO=0 rather than adding code to detect fedora that at some point will no longer be needed